src/Cooker/providers/GitProvider/getHashListFromHash.js

```diff
@@ -8,26 +8,30 @@ function getHashListFromHashToCommit(sha, commit) {
   }
 
   return new Promise((resolve, reject) => {
-    let hitPreviousRelease = false
+    let reachedHash = false
     const list = []
     commit
       .history()
       .on('commit', commit => {
-        if (hitPreviousRelease) {
+        if (reachedHash) {
           return
         }
 
         if (commit.sha() === sha) {
-          hitPreviousRelease = true
+          reachedHash = true
         } else {
           list.unshift(commit.sha())
         }
       })
       .on('end', () => {
-        if (hitPreviousRelease || list.length < 50) {
+        if (reachedHash || sha === 'Big Bang') {
```

Comment:
Git has this notion of "earliest ever thing" and they call it "Big Bang" in some places (hence the name). `getLatestReleaseHash` will return "Big Bang" if none found. It is also a way to say "give me all commits" (if we specify initial commit, it is ignored).

Since we throw an error on erroneous hash values (which are easy to come by as I realised today), we need something special that says "I did not forget to give you a hash, but walk the history all the way to the origin as if I had provided an invalid hash"...